download.html

```diff
@@ -38,6 +38,10 @@ <h3>Download Types</h3>
         <td><a href="https://github.com/nunit/nunit.xamarin/releases/latest">NUnit Xamarin Runners 3.6.1</a></td>
         <td class="text-right">March 14, 2017</td>
       </tr>
+      <tr>
+        <td><a href="https://github.com/nunit/dotnet-new-nunit/releases/latest">NUnit 3 Template for dotnet new CLI</a></td>
+        <td></td>
+      </tr>
```

Comment:
Perhaps we should add a date?

Reply:
Yeah, I didn't as then it would need updating - and I think @halex2005 was keen to avoid that from #77. It does mean it's inconsistent with the others on that page, but I think that's ok?

Reply:
I don't mind leaving out the date. I also noticed that some of the other entries e.g. the adapter is referencing old versions in the title and in the dates. Perhaps all items should just link to latest and not mention specific dates or version numbers (not that it much work to update these, but I think it is often forgotten).

Reply:
Yeah, I personally agree with you. I think it was a controversial idea, last time I suggested it however. 😛